app_with_external_db.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        RAILS_ENV: production
+    network_mode: host
```

Comment:
key to allow to connect to the external db via url

Reply:
Is this necessary if the base is on a different server?
I think this setting is needed to be able to use the host database from the container